data-structures/intro-to-data-structures.md

```diff
@@ -70,3 +74,4 @@ Using a timer, pair up for 5 minutes and quiz each other while reviewing the dif
 - [Watch Data Structures and Algorithms in action at visualgo.net](https://visualgo.net/en)
 - [Article w/Videos:  Data Structures and Algorithms in JavaScript by Beau Carnes](https://medium.freecodecamp.org/10-common-data-structures-explained-with-videos-exercises-aaff6c06fb2b)
 - [Article: https://www.tutorialspoint.com/data_structures_algorithms/data_structures_basics.htm](https://www.tutorialspoint.com/data_structures_algorithms/data_structures_basics.htm)
```

Comment:
Since tutorials point is in the lesson, please move the link from supplemental to “materials”.

Reply:
okayy! i've made the required changes. please check.